examples/siem-events/verify.sh

```
@@ -0,0 +1,327 @@
+if wait_for_health "$PROXY_PORT"; then
+  pass "pipelock healthy on 127.0.0.1:$PROXY_PORT"
+else
+  fail "pipelock did not become healthy"
+  tail -30 "$PROXY_LOG" >&2 || true
+  printf '\n\033[1m=== Results: %s passed, %s failed ===\033[0m\n\n' "$PASS" "$FAIL"
+  exit 1
+fi
+
+# -- Test 3 -------------------------------------------------------------------
+step "Test 3: clean fetch does not emit blocked event"
+BEFORE="$(event_count || echo 0)"
```

Shellcheck (0.11.0)

[warning] BEFORE appears unused. Verify use (or export if used externally).

(SC2034)